README.md

```diff
@@ -239,10 +95,6 @@ Make sure the secret is at least 30 characters and all random, no regular words
 or you’ll be exposed to dictionary attacks. You can use `rake secret` to
 generate a secure secret key.
 
-#### `SESSION_SECRET_KEY`
-
-This is used to sign the session used by the Sidekiq admin interface.
-
 #### `SMTP_USERNAME`, `SMTP_PASSWORD`
```

Comment:
I thought that it was already was dead code in public but I see it is being used by the visitors step model